### ISSUE TITLE
Bug: Ordering is missing in demo

### ISSUE THREAD

**Issue by ShmuelTreiger** [dzhuang/django-galleryfield#64]
Hi!

I'm running the demo and it seems the ordering is missing.

![image](https://github.com/user-attachments/assets/1d735673-a16b-46a4-995e-de20af22b56b)

Or else if I'm doing something wrong, please let me know. I just followed the instructions exactly.

**Comment by dzhuang** [comment]
Do you mean order by file names? That's not a feature of the package because Blueimp fileupload does not have this feature. For ordering, the related feature is about drag N drop to order the images.

**Comment by ShmuelTreiger** [comment]
Yes, I meant the drag and drop. The arrows on the right in your picture were missing:

https://raw.githubusercontent.com/dzhuang/django-galleryfield/main/demo/static/demo/screen_upload.png

Or were they removed and I can drag and drop without them?

**Comment by dzhuang** [comment]
Thanks for reporting this issue.

**Comment by corentinbettiol** [comment]
The arrows are not here anymore, but we found out that sortable is still working. You need to keep your click for a few hundreds milliseconds before being able to move the entries (there's a small delay).
